app/bundles/DynamicContentBundle/Views/DynamicContent/form.html.php

```diff
@@ -9,35 +9,188 @@
  * @license     GNU/GPLv3 http://www.gnu.org/licenses/gpl-3.0.html
  */
 
-$view->extend('MauticCoreBundle:FormTheme:form_simple.html.php');
+$view->extend('MauticCoreBundle:Default:content.html.php');
 $view->addGlobal('translationBase', 'mautic.dynamicContent');
 $view->addGlobal('mauticContent', 'dynamicContent');
+
```

Comment:
Lots of duplicate code to what's in the LeadBundle for lists. Can we not leverage templates  in order to not duplicate this code?

Reply:
There are a lot of differences between the two that's why I used a new one